modules/customizing-and-automating-your-compliance-scans.adoc

```diff
@@ -1,24 +1,27 @@
 // Module included in the following assemblies:
 //
-// * manage-compliance/managing-compliance-20.adoc
+// * operating/managing-compliance.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="customizing-and-automating-your-compliance-scans_{context}"]
 = Customizing and automating your compliance scans
 
 By creating a compliance scan schedule, you can customize and automate your compliance scans to align with your operational requirements.
 
+:FeatureName: Configuring scan schedules
+include::snippets/technology-preview.adoc[]
+
 .Procedure
 
-. In the {product-title-short} portal, go to the *Compliance (2.0) -> Cluster Compliance -> Schedules* tab.
+. In the {product-title-short} portal, go to *Compliance -> Schedules*.
```

Comment:
We need to add the ability to configure a report notifier in this section.  Additionally a mention of how to send a report for a scan schedule would be worth mentioning.

Reply:
+1